### ISSUE TITLE
dynamicaly placeholder changing is not work with "altinput" mode

### ISSUE THREAD

**Issue by neko2me** [mezoistvan/ng2-flatpickr#35]
sample code
https://codesandbox.io/s/yk87mmx9x

I am doing this now

``` ts
this.picker.flatpickrElement.nativeElement.getElementsByClassName("input")[0].placeholder = "...";
```

**Comment by AbuzerAsif** [comment]
@mezoistvan If you need help with this issue, then you can assign it to me. I am here to help...

**Comment by mezoistvan** [comment]
@AbuzerAsif I'm not working on the issue unless it is assigned to me, so feel free to work on it!

**Comment by AbuzerAsif** [comment]
@mezoistvan Here is the PR for the above issue https://github.com/mezoistvan/ng2-flatpickr/pull/60

**Comment by AbuzerAsif** [comment]
@mezoistvan During my testing i have also noticed the same issue happening for Ng2FlatpickrDirective as well. 

Here is the PR for it https://github.com/mezoistvan/ng2-flatpickr/pull/61

**Comment by mezoistvan** [comment]
Thank you so much! Released in v7.0.4 🎆
